src/app/lib/config/services/gujarati.ts

```diff
@@ -157,7 +157,7 @@ export const service: DefaultServiceConfig = {
                   'અમારી વેબસાઇટને યોગ્ય રીતે કાર્યરત રાખવા માટે સીમિત સ્તરે કેટલીક જાણકારીઓ સેવ કરવી પડે છે જેના માટે અમે આપની મંજૂરી નથી માગતા.',
                 para4: {
                   text: 'એ જરૂરી જાણકારીઓ વિશે વધારે વાંચો જે અમે આપની ડિવાઇસ પર સ્ટોર કર્યું છે જેથી અમારું પેજ આપને યોગ્ય રીતે દેખાય.',
-                  url: 'https://www.bbc.co.uk/usingthebbc/strictly-necessary-cookies/',
+                  url: 'https://www.bbc.com/usingthebbc/strictly-necessary-cookies/',
```

Comment:
These appear to 404 for me. Although the previous `.co.uk` one 404s also 🤔

Reply:
This one apprears to work: https://www.bbc.com/usingthebbc/cookies/strictly-necessary-cookies/

Reply:
Will take a look, thanks 👍

Reply:
Heya, soz I was looking at the wrong paragraph link, I've changed it to the correct one now :)

Reply:
Cool. I think some of the snapshots might need updating, but looks good to me!